x/params/types/subspace_test.go

```diff
@@ -13,28 +13,28 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/store"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/params/subspace"
 	"github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 )
 
 type SubspaceTestSuite struct {
 	suite.Suite
 
 	cdc codec.Marshaler
 	ctx sdk.Context
```

Comment:
not sure why we need a ctx on a types/ test suite. ctx belongs only to keepers

Reply:
I saw that when I was moving the packages, looks like Subspace has tests that uses other parts of the codebase (is not a unit test then), maybe we can get rid of it.